app/models/annotation.rb

```diff
@@ -37,6 +37,7 @@ def get_data(include_creator=false)
       content: annotation_text.content,
       annotation_category:
         annotation_text.annotation_category&.annotation_category_name,
+      category_id: annotation_text.annotation_category&.id,
```

Comment:
Sorry I missed this one. The name is fine, but in general I'd prefer using the exact attribute names when possible: here that would be `annotation_category_id` instead of `category_id`. In fact, you can access `annotation_text.annotation_category_id` explicitly. :)